cypress/support/commands.ts

```diff
@@ -38,6 +45,13 @@ export const pickWithContent = (
   return cy.contains(selectorString, content);
 };
 
+// Selects the input element with the given `data-cy` ID and pastes
+// the value inside
+export const pasteInto = (ids: string[], value: string): void => {
```

Comment:
I could not find `input` as a cypress command. 
Trying your example results in a compile error: `Property 'input' does not exist on type 'Chainable<JQuery<HTMLElement>>'.`

I prefer to have `pasteInto` in a separate command so that it is very visible what is happening. We should only use the command when entering the input is not part of the user experience we want to test.

Reply:
> I could not find `input` as a cypress command.

Sorry, I meant `.type(value)`.

> I prefer to have `pasteInto` in a separate command so that it is very visible what is happening. We should only use the command when entering the input is not part of the user experience we want to test.

That explains it, thanks.